pulsar-functions/runtime/src/main/java/org/apache/pulsar/functions/runtime/KubernetesRuntimeFactory.java

```diff
@@ -201,6 +201,11 @@ public void close() {
     @Override
     public void doAdmissionChecks(Function.FunctionDetails functionDetails) {
```

Comment:
can we add a unit test for this and are we sure after setupClient() appsClient and coreClient are not null